client/src/services/api.js

```diff
@@ -20,6 +26,18 @@ class Api {
     this.apiUrl = 'http://localhost:8080/api/';
   }
 
+
+  // eslint-disable-next-line no-unused-vars
+  getLogedin(_email, _password) {
+    axios.post('/authenticate', {
+      email: _email,
+      password: _password,
+
+    }).then((res) => {
+      localStorage.setItem('Bearer ', res.data);
```

Comment:
Don't save this logic in class Api

Reply:
why ?

Reply:
It is general class. It has only one responsibility - makes HTTP requests and sends response to component, which invokes this class method.
Your logic is connected to login page and only to login page. Your classes should be single responsibility

Reply:
allready removed